package.json

```diff
@@ -26,10 +26,13 @@
     "build:icons:glyphs": "npx @svgr/cli --expand-props none --template src/utils/svgToIconTemplate/index.js -d src/icons/ src/svgs/glyphs/",
     "build:icons:index": "create-index src/icons/",
     "build:rollup": "rollup -c",
-    "build:babel": "babel src -d lib",
+    "build:babel": "babel src -d lib --extensions '.js,.ts,.tsx'",
     "build:svgs:index": "create-index src/svgs/ --recursive --update --extensions svg",
     "publish-package": "run-s build && yarn publish",
-    "heroku-postbuild": "yarn run build:icons && yarn run build:icons:index && yarn run build-storybook"
+    "heroku-postbuild": "yarn run build:icons && yarn run build:icons:index && yarn run build-storybook",
+    "tsc": "tsc",
+    "tsc-with-tsconfig": "tsc --allowJs true --baseUrl src --declaration false --emitDeclarationOnly false --esModuleInterop true --experimentalDecorators true --forceConsistentCasingInFileNames true --jsx preserve --module commonjs --moduleResolution node --noEmit true --noImplicitReturns true --noUnusedLocals true --noUnusedParameters true --skipLibCheck true --strict true --suppressImplicitAnyIndexErrors true --target es2018",
```

Comment:
We are probably deprecating this in the future, in favor of just `tsc`, FYI. The consideration is about how much of the codebase is converted. This command works well for spotting issues in the specific files being committed, but it's possible that changes in one file creates issues in other files that aren't being edited, so this command cannot catch those.

Reply:
but you said the `tsc` command alone wont pick up the tsconfig.json in the hook phase? Yeah I understood this is to make the lint-staged hook work. this is literally copied this from pocketderm anyway

Reply:
solved updating lint-staged config